### PR TITLE
Android cakes parallel -- Changes from Priyank

### PR DESCRIPTION
**Pull request by priyankshah217**
Hi Sai,

Kindly review changes and merge it.

Thanks,
Priyank

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -176,6 +192,28 @@ Your should see report file generated as ExtentReport.html under the target fold
 ##WIP
 iOS WebTest using Safari
 
+##FAQ
+Q. Is this framework supports to run multiple IOS simulators?
+
+A. No, Apple does not allow to do so.
```
Comment (saikrishna321) [inline README.md]: @priyankshah217 shouldn't this be Appium does not allow ?

Comment (SrinivasanTarget) [inline README.md]: I think facebook does this successfully

Comment (saikrishna321) [inline README.md]: @SrinivasanTarget yup they do, so ideally appium would also get this in up coming releases with help of WebDriverAgent

Comment (SrinivasanTarget) [inline README.md]: Yes

Comment (saikrishna321) [inline README.md]: @priyankshah217 thoughts ?

Comment (priyankshah217) [inline README.md]: Yes, using webdriveragent this is possible. But i guess with appium-webdriveragent is still in beta release. So we can say multiple emulator will be supported in appium's future releases.

**Pull request merged**